app/Models/User.php

```diff
@@ -478,6 +478,11 @@ public static function lookupWithHistory($usernameOrId, $type = null, $findAll =
             return $user;
         }
 
+        // don't perform username change history lookup if we're searching by ID
+        if ($type === 'id') {
+            return null;
+        }
```

Comment:
I don't see how this will ever be true considering `$type` is passed as null everywhere...? :thinking:

Reply:
Problem for the future 🤔